catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/app/WhenStartedBrickTest.java

```diff
@@ -85,17 +76,4 @@ public void whenStartedBrick() {
 		onBrickAtPosition(1).checkShowsText(R.string.brick_wait);
 	}
 
-	public void addBrick(Class<?> brickHeaderClass, int brickCategoryId) {
```

Comment:
id prefer to leave this here.
- it is only used in this test class, theres already ~100 brick test classes that work without something like this, and once it ends up in a utility method, people will misuse this for adding bricks instead of writing proper setup methods.

Reply:
so you are saying its fine to have a duplicate method in my test case and the WhenStartedBrickTest
 
I'm asking this since on my last pull request, which was also about this test case, you told me that I should remove the duplicates from these two so I am confused a little bit. Or do you want me to access the method within the WhenStartedBrickTest from SetVariableTest?

Reply:
ah, so you use this one aswell. then its okish... 
i thought theres just one class using it....

please rename to addBrickViaUi or something...